index.js

```diff
@@ -240,7 +240,9 @@ export default class Supercluster {
     }
 
     _limitZoom(z) {
-        return Math.max(this.options.minZoom, Math.min(+z, this.options.maxZoom + 1));
+        return Math.floor(
+            Math.max(this.options.minZoom, Math.min(+z, this.options.maxZoom + 1))
+        );
```

Comment:
```suggestion
        return Math.max(this.options.minZoom, Math.min(Math.floor(+z), this.options.maxZoom + 1));
```

Reply:
Could we use `Number.parseInt(z, 10)` directly in this case?

Reply:
Yeah, not a fan of `parseInt`, and generally not a fan of excessive validation of arguments in JS libraries. `+z` is the extent I can go to for input coercion.